packages/@n8n/db/src/migrations/common/1750252139168-LinkRoleToUserTable.ts

```diff
@@ -0,0 +1,79 @@
+import type { MigrationContext, ReversibleMigration } from '../migration-types';
+
+/*
+ * This migration
+ */
+
+export class LinkRoleToUserTable1750252139168 implements ReversibleMigration {
+	async up({
+		schemaBuilder: { addForeignKey, addColumns, column },
+		escape,
+		runQuery,
+	}: MigrationContext) {
+		const tableName = escape.tableName('role');
+		const userTableName = escape.tableName('user');
+		const slugColumn = escape.columnName('slug');
+		const roleColumn = escape.columnName('role');
+		const roleSlugColumn = escape.columnName('roleSlug');
+		const roleTypeColumn = escape.columnName('roleType');
+		const systemRoleColumn = escape.columnName('systemRole');
+
+		// Make sure that the global roles that we need exist
+		try {
```

Comment:
DO you think it would make sense to have a loop here (of ['global:owner', ''global:admin', 'global:member']) ?